### DISCUSSION TITLE
Help Needed - "sh: 17: git: not found"

### DISCUSSION THREAD

**Discussion by hdent1232** [dominiksalvet/asus-fan-control#140] [Q&A] [answered]
Hi, sorry if this is the wrong place to post this, but I just switched to linux 2 days ago after windows was causing me too many problems and I'm having trouble figuring out how to fix this issue. I really need a way to control my fan speed as the other solutions I've tried dont pick up my fan and my laptop is always getting dangerously hot. I've ran the commands exactly as shown in the installation guide but for whatever reason I keep getting this error and can't install Asus Fan Control. Went over to the WGet github page and ran all the installation commands there too, any help would be greatly appreciated

**Comment by dominiksalvet** [comment, marked as answer]
Hello, you need to install `git` first. If you are using a Debian-based Linux distro, use `sudo apt install git`. If you are using a different Linux distro, find your command online. Then, it should not end up with this error.

**Reply by hdent1232** [reply]
thank you so much!
